blueocean-pipeline-api-impl/src/main/java/org/jenkinsci/plugins/workflow/job/views/GraphDumpAction.java

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
This is a debugging helper, built using GraphVizAction as a template. I'm leaving it here for now, but I'll talk to Pipeline see if they want it.